app/views/comments/_form.html.erb

```diff
@@ -0,0 +1,9 @@
+<h4>Post a Comment</h4>
+
+<%= form_for @comment do |f| %>
+
+  <%= f.hidden_field :job_id, :value => @job.id %>
```

Comment:
we already talked about how nesting our routes would help us avoid the need to send hidden data 👍